Furball.Vixie.Backends.OpenGL.Shared/TextureGL.cs

```diff
@@ -79,7 +81,7 @@ public class TextureGL : Texture, IDisposable {
         /// <summary>
         /// Used for determening whether or not to Flip it internally because it's a framebuffer
         /// </summary>
-        internal bool IsFramebufferTexture;
+        public bool IsFramebufferTexture;
```

Comment:
this should defnitly not be opened up to the end user to change however they like, internal fits better here.